src/Sylius/Behat/Context/Setup/ZoneContext.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Setup;
+
+use Behat\Behat\Context\Context;
+use Sylius\Bundle\AddressingBundle\Factory\ZoneFactoryInterface;
+use Sylius\Bundle\SettingsBundle\Manager\SettingsManagerInterface;
+use Sylius\Component\Addressing\Model\ZoneInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class ZoneContext implements Context
+{
+    /**
+     * @var array
+     */
+    private $euMembers = [
+        'BE', 'BG', 'CZ', 'DK', 'DE', 'EE', 'IE', 'GR', 'ES',
+        'FR', 'IT', 'CY', 'LV', 'LT', 'LU', 'HU', 'MT', 'NL',
+        'AT', 'PL', 'PT', 'RO', 'SI', 'SK', 'FI', 'SE', 'GB',
+    ];
+
+    /**
+     * @var RepositoryInterface
+     */
+    private $zoneRepository;
+
+    /**
+     * @var SettingsManagerInterface
+     */
+    private $settingsManager;
+
+    /**
+     * @var ZoneFactoryInterface
+     */
+    private $zoneFactory;
+
+    /**
+     * @param RepositoryInterface $zoneRepository
+     * @param SettingsManagerInterface $settingsManager
+     * @param ZoneFactoryInterface $zoneFactory
+     */
+    public function __construct(RepositoryInterface $zoneRepository, SettingsManagerInterface $settingsManager, ZoneFactoryInterface $zoneFactory)
+    {
+        $this->zoneRepository = $zoneRepository;
+        $this->settingsManager = $settingsManager;
+        $this->zoneFactory = $zoneFactory;
+    }
+
+    /**
+     * @Transform :taxZone zone
+     * @Transform zone :taxZone
+     */
+    public function castZoneCodeToZone($taxZone)
```

Comment:
I am wondering about naming of these methods, why not just ``getZoneByCode``? This way we can use those naturally in non-step methods.